portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,57 @@
+body{
```

Comment:
From a formatting standpoint, prefer to have a single space before opening curly braces. So:
```
body {
  ...
}
```

It's also consistent with the existing `#content` style :)